src/music_mix.cpp

```diff
@@ -191,8 +195,18 @@ void KMusic::play_sample(void* chunk, int, int, int, int)
     }
     Mix_PlayChannel(-1, reinterpret_cast<Mix_Chunk*>(chunk), 0);
 }
-void KMusic::set_volume(float, int)
+/* \brief Set overall sound volume.
+ * \param   volume 0 (silent) to 250 (loudest)
+ */
+void KMusic::set_volume(int sound_volume)
```

Comment:
Are music or sound volumes dependent on each other?

In other words, can I adjust the music volume without changing the sound volume, or adjust the sound volume without the music volume changing?

If this adjusts only sounds, could this be named `set_sound_volume`? If it adjusts both, could it be named something like `set_system_volume` or `set_overall_volume` or something?

Reply:
It was always supposed to - see 
https://github.com/OnlineCop/kq-fork/blob/8c638937d127b1bb705ea07682fdbb849fb32087/src/setup.cpp#L148
It means you can't have quiet sounds and loud music, would anyone want that?
In terms of the name, I'm not bothered. The code comment says it's 'overall sound volume' so I think it's clear.